lib/glue/tasks/bandit.rb

```diff
@@ -18,7 +18,13 @@ def initialize(trigger, tracker)
 
   def run
     rootpath = @trigger.path
-    @result=runsystem(true, "bandit", "-f", "json", "-r", "#{rootpath}")
+    context = SecureRandom.uuid
+    @tmppath = "/tmp/#{context}/"    
```

Comment:
Maybe use [`tmpdir`](https://ruby-doc.org/stdlib-1.9.3/libdoc/tmpdir/rdoc/Dir.html) for multi-platform support? Also, might be nice to allow the user to override the location.
Also, are all users allows to write there?